src/controllers/deploy.go

```diff
@@ -69,7 +69,11 @@ func internaldeploy(a *history.ActionInstance) ([]byte, error) {
 		state.Access = a.Access
 		state.Server = a.Server
 		state.Status = "deploying"
-		history.SetState(a.RepoURL, state)
+		if err1 := history.SetState(a.RepoURL, state); err1 != nil {
+			log.Infof("subdomain(%s) in use", a.RepoURL)
+			output = []byte("Subdomain is already in use. Deploy with a different subdomain.")
```

Comment:
Same here. Return a better error message and format it in Camel Case and return as output.